pyomo/opt/testing/pyunit.py

```diff
@@ -32,13 +32,16 @@ def _failIfPyomoResultsDiffer(self, cmd=None, baseline=None, cwd=None):
             baseline = "\n".join(INPUT.readlines())
             INPUT.close()
     
-        output = pyutilib.subprocess.run(cmd)
+        output = subprocess.run(cmd, stdout=subprocess.PIPE,
+                                stderr=subprocess.PIPE)
     finally:
         os.chdir(oldpwd)
     
-    if output[0] != 0:
+    if output.returncode != 0:
         self.fail("Command terminated with nonzero status: '%s'" % cmd)
-    results = extract_results(re.split('\n',output[1]))
+    # !!THIS SEEMS LIKE A BUG!! - mrmundt #
+    # There is nothing imported called "extract_results" nor "compare_results"
+    results = extract_results(re.split('\n', output.stdout.decode("utf-8")))
```

Comment:
I agree.  I not sure this code is / can be run?  I didn't find an extract_results anywhere in either Pyomo or PyUtilib.  I wonder if this entire directory (`pyomo/opt/testing`) is actually abandoned code?